src/msw/listbox.cpp

```diff
@@ -611,6 +614,11 @@ void wxListBox::SetHorizontalExtent(const wxString& s)
     //else: it shouldn't change
 }
 
+void wxListBox::MSWSetTabStops(const wxArrayInt& tabStops)
+{
```

Comment:
I wonder if adding something like
` wxCHECK_RET(!tabStops.empty(), "tabStops can't be empty");` 
to avoid dereferencing possibly invalid `tabStops.begin()` would be of any use...

And a supernitpick, but I would prefer using `size()` instead of` Count()`...

Reply:
I'll apply both you suggestions and will also change `wxArrayInt` to `wxVector<int>` as @vadz mentioned.
Thank you for the review!